tripcode_test.go

```diff
@@ -11,25 +11,33 @@ func TestGenerateSalt(t *testing.T) {
 		"!@#$%^&*()": "G.",
 		"f}E":        ".E",
 		"©":          "H.",
-		"訛":          "H.",
+		"訛":         "H.",
 		"'":          "H.",
 	}
 	for pass, expected := range cases {
 		salt := generateSalt(pass)
 		if expected != salt {
-			t.Error("Expected "+expected+", got", salt)
+			t.Error("Expected \"%s\", got \"%s\"", expected, salt)
 		}
 	}
 }
 
+func TestSecureTripcode(t *testing.T) {
+	expect := "PqG4A0fkUs"
+	trip := SecureTripcode("pass", "salt")
+	if trip != expect {
+		t.Errorf("SecureTripcode: expected \"%s\", got \"%s\"", expect, trip)
+	}
+}
+
 func TestTripcode(t *testing.T) {
 	cases := map[string]string{
 		"asd":        "TAPy3blMsc",
 		"adasd":      "IOuORdzMKw",
 		"!@#$%^&*()": "BpZUCmJAIQ",
 		"f}E":        "oUBoOTrysY",
-		"©":          "",
-		"訛":          "c8eDXvwFLQ",
+		"©":          "hEp4vbueZo", // Should be nothing?
```

Comment:
Is there any other source for this claim? Both [minichan](https://minichan.org/triptest?name=%23%C2%A9) and [desktopthread](http://desktopthread.com/tripcode.php) tell me that a lone `©` should equal to a trip `cPUZU5OGFs` without variations. Not that I yet know how this is achieved.